src/field/arena.ts

```diff
@@ -59,7 +55,7 @@ export class Arena {
     this.biomeType = biome;
     this.tags = [];
     this.bgm = bgm;
-    this.trainerPool = biomeTrainerPools[biome];
+    this.trainerPool = allBiomes.get(biome).trainerPool;
```

Comment:
The constructor param should probably be renamed as `biome` would be associated with the class and not an id